src/pid4cat_model/schema/pid4cat_model.yaml

```diff
@@ -208,16 +210,17 @@ slots:
     description: The field that was changed
 
   # Slots for Agent
-  name:
+  agent_name:
     slot_uri: schema:name
     description: The name of the agent
-  contact_information:
+  agent_email:
     slot_uri: schema:email
     description: Identification of the agent that registered the PID, with
       contact information. Should include person name and affiliation, or position
       name and affiliation, or just organization name. e-mail address is preferred
       contact information.
-  person_orcid:
+      TODO: should be better defined - to be clearly parsable by machines
```

Comment:
Yes! This should also have the email-regex as above for curation_contact `pattern: "^\\S+@[\\S+\\.]+\\S+"`
Feel free to add it already now.